app/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -39,3 +39,29 @@ $sizes: (
 .font-weight-semibold {
   font-weight: 600;
 }
+
+.positive-significance {
+  color: green;
+  font-weight: bold;
+}
+
+.negative-significance {
```

Comment:
Can we use Bootstrap's `text-danger font-weight-bold` classes for this instead, same with `positive-significance` using `text-success font-weight-bold` instead?